js-src/aws/MetadataUpload.ts

```diff
@@ -0,0 +1,65 @@
+
+interface TdrFile extends File {
+    webkitRelativePath: string;
+}
+// Not uploading data just logging data to prove File api works
+export const uploadFileMetadata = (files:FileList) => {
+
+    const uploads = Array.from(files).map(async file => {
+        await getFileMetadata( file.name, file)
+    });
+
+    return Promise.all(uploads);
+
+};
+
+const getFileMetadata =  async ( name:String, content:File) => {
+    const fileInfo = await getFileInfo(<TdrFile>content);
+    console.log(fileInfo);
+    return fileInfo
+
+};
+
+const hexString = (buffer:ArrayBuffer) => {
+    const byteArray = new Uint8Array(buffer);
+
+    const hexCodes = [...byteArray].map(value => {
+        const hexCode = value.toString(16);
+        return hexCode.padStart(2, "0");
+     });
+
+    return hexCodes.join("");
+};
+
+
+const generateHash = (file:File) => {
+    const crypto = self.crypto.subtle;
```

Comment:
How is the crypto library being loaded? I don't see any import statements or changes to package.json.

Reply:
Ah, it's because it's not [crypto-js](https://www.npmjs.com/package/crypto-js). This is just calling [Window.crypto](https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto).

As far as I know, that's fine - can I just check it's what you were expecting?